app/src/main/res/values/strings.xml

```diff
@@ -495,4 +495,10 @@
     <string name="server_setup_incomplete">The setup of this server has not been completed. Open Jellyfin in a web browser to finish setup before signing in.</string>
     <string name="episode_name_special">special</string>
     <string name="app_notification_beta">App updated to %1$s\nThank you for participating in the Jellyfin beta program</string>
+    <string name="item_deleted">%1$s deleted</string>
+    <string name="item_deletion_failed">Unable to delete %1$s</string>
+    <string name="pref_enable_media_management">Enable media management</string>
+    <string name="pref_enable_media_management_description">Show option to manage media while browsing</string>
+    <string name="item_delete_confirm_title">Delete item</string>
+    <string name="item_delete_confirm_message">Are you sure you want to delete this item? This action cannot be undone.</string>
```

Comment:
I don't know what the message used to say, but maybe make it clear that this will remove the file on disk?

Reply:
Hardcoded string was
`This will PERMANENTLY DELETE " + mBaseItem.getName() + " from your library.  Are you VERY sure?`

Reply:
@nielsvanvelzen & @crobibero:
Sorry, I didn't see these comments before the PR got merged, but what do you think of this?
```suggestion
    <string name="item_delete_confirm_message">Are you sure you want to delete this item from the server? This action cannot be undone.</string>
```

Reply:
Yeah that's a good suggestion, made a PR #2477